docs: updates (#1587)
terramate-io/terramate: docs/cli/orchestration/order-of-execution.md

---
title: Order of Execution

> description: Define the order stacks are executed in.

Reviewer: ```suggestion
description: Learn about the order of execution when orchestration stacks with Terramate.
```

> outline: [2, 3]

Reviewer: Shall we add this as default settings to the config? https://vitepress.dev/reference/default-theme-config#outline

Author: does not matter fmpov.

---

# Order of Execution

## Implicit Order of Execution

> implicit order will be detected by Terramate depending on integrations and hierarchy of stacks.

Reviewer: ```suggestion
Implicit order will be detected by Terramate depending on integrations and hierarchy of stacks.
```

### Parent and Child Stacks

Currently Terramate orders stacks implicitly when they are nested.

> Nested Stacks are stacks that are in a subdirectory of stack.

Reviewer: ```suggestion
nested stacks are stacks that are in a subdirectory of stack.
```
